Dockerfile

```diff
@@ -0,0 +1,38 @@
+# See examples/README.md to see how to build and run this
+
+FROM ubuntu:16.04
+
+# Uncomment the next 2 lines if you're behind a proxy:
+# ENV http_proxy="http://proxy.example.com:8080"
+# ENV https_proxy=${http_proxy}
```

Comment:
Since we're usually connected to the appliances network through a VPN, when using vagrant for instance, we often use no_proxy to list exactly the appliance's IPs so those are accessible, otherwise just external sites become accessible with the http/https proxy. 
Assuming the same for docker.